src/app/components/normal-component/header/header.component.spec.ts

```diff
@@ -53,7 +53,7 @@ describe('HeaderComponent', () => {
         it('should remove comment in header when disconnecting', fakeAsync(async() => {
             // Given a connected user that has an observedPart
             ConnectedUserServiceMock.setUser(UserMocks.CONNECTED_AUTH_USER);
-            const observedPart: FocusedPart = FocusedPartMocks.CANDIDATE;
+            const observedPart: ObservedPart = ObservedPartMocks.CANDIDATE;
```

Comment:
C'était pas mieux comme nom "Focused" justement, car on est pas observeur dessus mais c'est celle sur laquelle le "focus" "applicatif" est ?

Reply:
faut choisir l'un ou l'autre, les deux sont mauvais je dirais.
T'es pas focused dessus puisque tu peux être sur une autre page.
T'observe pas car t'es pas observeur.
T'es "in game", et c'est la seule partie où tu peux être. Une idée de meilleur non ?

Reply:
courant veut rien dire vraiment, c'est le jeu ou tu fait quoi couramment avec ? c'est un peu vide de sens, alors que si t'omet jouer sur les mots de "jouer ce n'est pas observer" ou "être abonné si t'es pas sur la page t'as pas d'abonnement avec" ceux là sont meilleurs.

Sinon uniqueGame ou linkedGame pourraient marcher, c'est l'unique jeu auquel tu peux être lié (jusqu'à le quitter pour un non-joueur, ou à le finir pour un joueur) ?

Reply:
bah, courant aka currentGame, le jeu auquel tu joue présentement
Quid de `GameInPlay` ? `UniqueGame` pourrait aussi passer ouais

Reply:
bah, courant aka currentGame, le jeu auquel tu joue présentement
Quid de `GameInPlay` ? `UniqueGame` pourrait aussi passer ouais

Reply:
pour moi focused était bien car il centralisais "unique"et pas forcément observé/joué (car c'p-e pas ça que tu fait)
ExclusiveCurrentGame ? CurrentFocussedGame ?
Faudrait un mot pour "que tu observe ou joue ou crée ou tente de joindre"

Reply:
mais focus = là où tu regarde / ce qui est devant, donc ça correspond pas à ce qu'on veut.
`OnlyCurrentGame` ? Et je réitère le `UniqueGame`
`ExclusiveCurrentGame` comme middle ground ça pourrait passer, mais pas trop fan.
Quid de simplement `CurrentGame`: t'as un current game, no matter ce que t'y fait.

Reply:
Only jouant le rôle "exclusive" je suis preneur, Unique aussi mais alors UniqueCurrentGame alors !
mais ouais pourquoi pas CurrentGame, clairement !

Reply:
Renommé en `CurrentGame` !